src/assets/scss/post/main.scss

```diff
@@ -23,62 +23,134 @@ li {
   }
 }
 
-%edit-no-outline {
-  color: $text-2;
-  background-color: $white;
-  padding: 0.25rem 1rem;
+%normal-btn {
```

Comment:
jusy a question, what doesn % mean?

Reply:
It is a placeholder selector that the content can be included in other classes using the `@extend` keyword.
The content enclosed in % the selector will not appear in the output CSS if it is not `@extend`ed.